### PR TITLE
Demo automated tests

### PR DESCRIPTION
**Pull request by ghost**
@shankari  working to get build of images created.

### CODE REVIEW

File: docker-compose.admin-panel.yml [hunk 1]
```diff
@@ -2,12 +2,12 @@ version: "3.6"
 
 services:
   mqtt-server:
-    image: ghcr.io/shankari/everest-demo/mqtt-server:0.0.1
+    image: ghcr.io/everest/everest-demo/mqtt-server:0.0.1
```
Comment (shankari) [inline docker-compose.admin-panel.yml]: Let's keep this in the `US-JOET` organization until the new image building process is in place

```suggestion
    image: ghcr.io/us-joet/everest-demo/mqtt-server:build_pkg_task_1_features
```

File: docker-compose.admin-panel.yml [hunk 2]
```diff
@@ -2,12 +2,12 @@ version: "3.6"
...
     logging:
       driver: none
 
   manager:
-    image: ghcr.io/shankari/everest-demo/manager:0.0.2
+    image: ghcr.io/everest/everest-demo/manager:0.0.2
```
Comment (shankari) [inline docker-compose.admin-panel.yml]: Ditto

File: mosquitto/Dockerfile [hunk 3]
```diff
@@ -2,4 +2,4 @@ FROM eclipse-mosquitto:2.0.10
 
 COPY mosquitto.conf /mosquitto/config/mosquitto.conf
 
-LABEL org.opencontainers.image.source=https://github.com/shankari/everest-demo
+LABEL org.opencontainers.image.source=https://github.com/everest/everest-demo
```
Comment (shankari) [inline mosquitto/Dockerfile:5]: This also needs to be `us-joet` for now until we have the image pipeline, and all the related permissions, integrated into everest

File: manager/tests/startup_tests.py
Comment (shankari) [inline manager/tests/startup_tests.py:1]: just to make sure that i understand - this is basically only testing `config-sil.yaml` with a happy path and an error path (the `diode_failure`, right?

File: manager/tests/startup_tests.py [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+
+    probe = ProbeModule(session)
+
+    if everest_core.status_listener.wait_for_status(10, ["ALL_MODULES_STARTED"]):
+        everest_core.all_modules_started_event.set()
+        logging.info("set all modules started event...")
+
+    charging_session_cmd = {'value': 'sleep 1;iec_wait_pwr_ready;sleep 1;draw_power_regulated 16,3;sleep 5;pause;sleep 5;diode_fail;sleep 5'}
+    expected_events = ['TransactionStarted', 'ChargingStarted','ChargingPausedEV', {'error': 'CarDiodeFault'}]
+
```
Comment (shankari) [inline manager/tests/startup_tests.py]: Again, for my understanding, the main change here, over the `startup_tests.py` in `everest-core` is that `everest-core` has a set of hardcoded commands. Here, after initializing the modules, we also send the sequence of commands and expect a sequence of events (with the corresponding changes to `ProbeModule`, of course).

**Review by shankari — CHANGES_REQUESTED** [review]
The test itself is fine. But all the images here needs to be us-joet until we finally start hosting the images on everest.

File: docker-compose.admin-panel.yml (hunk 1 above)
Comment (ghost) [reply docker-compose.admin-panel.yml]: @shankari I was planning to updated the remaining in their respective branches.

File: manager/tests/startup_tests.py
Comment (ghost) [reply manager/tests/startup_tests.py:1]: That is correct.

File: manager/tests/startup_tests.py (hunk 4 above)
Comment (ghost) [reply manager/tests/startup_tests.py]: The biggest change is that we were only getting high level events back. I updated the array to pass back either the event name or a dict with more details. This allows for deeper assertions in the test. There is way more that could probably be done but this proved we could inspect more using the stand alone probe module.

File: docker-compose.admin-panel.yml (hunk 1 above)
Comment (shankari) [reply docker-compose.admin-panel.yml]: @couryrr-afs  can you revert the extraneous changes then? I don't want to have a lot of flip-flopping that is merged, and it clutters up the review.

I am fine with merging once extraneous changes have been cleaned up/removed.

**Comment by drmrd** [comment]
Closing in favor of EVerest/everest-demo#5

**Pull request closed without merging**